src/go_quic_server_session.h

```diff
@@ -17,6 +17,7 @@
 #include "net/quic/quic_crypto_server_stream.h"
 #include "net/quic/quic_protocol.h"
 #include "net/quic/quic_session.h"
+#include "net/quic/quic_data_stream.h"
 
 namespace net {
 
```

Comment:
Since I can't comment on https://github.com/devsisters/goquic/pull/4/files#diff-76db07ddeed7d521ba4c7e0381799faaR52 due to Github restrictions, I'll comment here.

`GoQuicServerSession` should now inherit from `QuicSpdySession` due to underlying code changes.